src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/OppdaterVilkårsvurderingTest.kt

```diff
@@ -219,82 +221,6 @@ class OppdaterVilkårsvurderingTest {
         Assertions.assertTrue(nyInitBosattIRiketVilkår.single().resultat == Resultat.IKKE_VURDERT)
     }
 
-    @ParameterizedTest
-    @EnumSource(value = BehandlingType::class, names = ["FØRSTEGANGSBEHANDLING", "REVURDERING"])
-    fun `kopierResultaterFraForrigeBehandling skal kun ta med oppfylte og ikke aktuelle perioder såsant det ikke er ny førstegangsbehandling`(
```

Comment:
Erstattes av cucumbertesten `Ved kopiering av vilkårresultater skal avslag og opphør for barnehageplassvilkåret beholdes fra forrige behanlding`